src/eth/TransactionManager.js

```diff
@@ -210,18 +210,18 @@ class Tracker {
     return txs[0];
   }
 
-  clearExpiredListeners() {
-    each(this._transactions, (val, key) => {
-      val.forEach(tx => {
-        // TODO: check Tx age for 5 minutes or older:
-        // const age = (new Date() - new Date(tx._timeStampMined)) / 1000;
-        if (tx.isFinalized() || tx.isError()) {
-          log(
-            `deleting key ${key}, contract: ${tx.metadata.contract}, method: ${
-              tx.metadata.method
-            }`
-          );
-          delete this._transactions[key];
+  clearExpiredTransactions() {
+    each(this._transactions, (txList, key) => {
+      txList.forEach(tx => {
+        const txAge =
+          new Date(tx._timeStampMined).getMinutes() - new Date().getMinutes();
```

Comment:
This won't handle e.g. going from 1:57 to 2:02 -- just use `.getTime()` and divide by 60000 to convert from milliseconds to minutes